README.md

```diff
@@ -6,13 +6,13 @@ Cloudfuse uses file caching to provide the performance of local storage, or you
 Cloudfuse is a fork of [blobfuse2](https://github.com/Azure/azure-storage-fuse), and adds S3 support, a GUI, and Windows support.
 Cloudfuse supports clouds with an S3 or Azure interface.
 
-## Installation

```

Comment:
Could you keep the installation header and then have the smaller headers for Windows and Linux.